packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/runtime/DesktopConfigurator.java

```diff
@@ -61,8 +61,8 @@ public void refresh()
 
                 /** Desktop-specific config overrides */
                 pair( auth_store.name(), new File( dbDir, "./dbms/auth" ).getAbsolutePath() ),
-                pair( tls_certificate_file.name(), new File( dbDir, "./certificates/snakeoil.cert" ).getAbsolutePath() ),
-                pair( tls_key_file.name(), new File( dbDir, "./certificates/snakeoil.key" ).getAbsolutePath() ),
+                pair( tls_certificate_file.name(), new File( dbDir, "./certificates/neo4j.cert" ).getAbsolutePath() ),
+                pair( tls_key_file.name(), new File( dbDir, "./certificates/neo4j.key" ).getAbsolutePath() ),
```

Comment:
Do you really need to override the entire locations here? Can't you just provide a value for `dbms.directories.certificates`? In fact, isn't that then the default value which can be left out altogether?